### ISSUE TITLE
Fix Nan loss

### ISSUE THREAD

**Issue by makaichi** [huyanxin/phasen#11]
I got "Nan"  when use Mix loss to train (not speech denoise task), and Fix it by adding grad clip as fellows:

loss.backward()
nn.utils.clip_grad_norm_(self.estimator.parameters(), 10.0) # add this to clip grad
self.optimizer.step()

**Comment by huyanxin** [comment]
I think nan caused by other part and The grad clip had been already added in
https://github.com/huyanxin/phasen/blob/31ee2f1ba89b535142a5189abf913e9ac7f36404/steps/run_phasen.py#L76

**Comment by huyanxin** [comment]
I add eps in mixed loss, you can try it https://github.com/huyanxin/phasen/commit/929fa464e610b73c04fd85f5a4dcb77ec040fc30
